bitbybit-tests/src/bitenum_tests.rs

```diff
@@ -191,7 +191,7 @@ fn enum_with_64bits() {
 
 #[test]
 fn enum_with_implicit_discriminants() {
-    #[bitenum(u8, exhaustive = false)]
```

Comment:
I meant that the test should include both u8 AND u4 (meaning, we should have 2 tests) to ensure they both work going forward